CHANGELOG.adoc

```diff
@@ -21,17 +21,17 @@
 == Release 1.2 (2016-10-19)
 
 * https://github.com/vjuranek/radargun-plugin/issues/58[#58] Added Script source deprecation warning
-* https://github.com/vjuranek/radargun-plugin/issues/52[#52] Removed deprecation from JVM option on master node
```

Comment:
not sure if it worth to change it in the changelog, but no problem with it

Reply:
Yeah, let me take these changes out